brightlayer-ui/react-native-component-library
# Update Readme docs to contain all components and point to master

Fixes #238 and #239.


#### Changes proposed in this Pull Request:
- Update readme to point to master branch
- Update component readme to include all available components



#### To Test:
- Check the readmes... click the links... make sure they work and point to `master` branch
